### DISCUSSION TITLE
While running the karavan app in the Ubuntu Instance I am able to create the project but after forcebly stop the karavan in the terminal  and start again the created project is not  present . please check.

### DISCUSSION THREAD

**Discussion by kondetiramanakumar** [apache/camel-karavan#1273] [Q&A] [answered]
### Ask a question

While running the karavan app in the Ubuntu Instance I am able to create the project but after forcebly stop the Instance in the terminal  and start again the created project is not  present . please check.

able to run the karavan-app in the ubuntu Instance and able to create the create the project:

<img width="745" alt="image" src="https://github.com/apache/camel-karavan/assets/14121796/870b73bb-5f05-49ea-ba41-0bd8b7570f33">

After stopping the runnable instance
<img width="583" alt="image" src="https://github.com/apache/camel-karavan/assets/14121796/952e57a2-86c3-4fd6-9ca5-36ee5649bd47">

May I know why my created project is not present? any idea??
<img width="749" alt="image" src="https://github.com/apache/camel-karavan/assets/14121796/e6a08337-cd7b-45ee-9e35-2a29de5f52a7">

### Variant

None

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

None

**Comment by mgubaidullin** [comment, marked as answer]
This is normal behaviour. Karavan keeps all project data in memory until committed/pushed to git.
